[2025-03-22 14:58:41,801] INFO _log() L97   [33mPress CTRL+C to quit[0m - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:58:41,807] INFO _log() L97    * Restarting with stat - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:58:44,242] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 14:58:44] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:58:44,460] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 14:58:44] "[36mGET /openapi/swagger/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:58:44,538] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 14:58:44] "[36mGET /openapi/swagger/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:58:44,538] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 14:58:44] "[36mGET /openapi/swagger/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:58:44,820] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 14:58:44] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:58:44,829] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 14:58:44] "[36mGET /openapi/swagger/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:58:50,418] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 14:58:50] "GET /rescue-point HTTP/1.1" 200 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:58:57,064] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 14:58:57] "DELETE /rescue-point?id=1 HTTP/1.1" 200 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:59:03,858] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 14:59:03] "GET /rescue-point HTTP/1.1" 200 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:02:24,400] INFO _log() L97   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:02:24,401] INFO _log() L97   [33mPress CTRL+C to quit[0m - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:02:24,404] INFO _log() L97    * Restarting with stat - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:02:27,342] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 22:02:27] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:02:27,692] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 22:02:27] "[36mGET /openapi/swagger/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:02:27,831] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 22:02:27] "[36mGET /openapi/swagger/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:02:27,846] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 22:02:27] "[36mGET /openapi/swagger/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:02:28,002] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 22:02:28] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:02:28,035] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 22:02:28] "[36mGET /openapi/swagger/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:02:43,202] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 22:02:43] "[31m[1mPOST /generate-rescue-point HTTP/1.1[0m" 422 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:03:18,014] INFO _log() L97    * Detected change in 'D:\\Desenvolvimento\\PUC\\rescue-api\\schemas\\rescue_point.py', reloading - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:03:18,193] INFO _log() L97    * Restarting with stat - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:03:21,428] INFO _log() L97    * Detected change in 'D:\\Desenvolvimento\\PUC\\rescue-api\\schemas\\rescue_point.py', reloading - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:03:21,585] INFO _log() L97    * Restarting with stat - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:03:26,845] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 22:03:26] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:03:27,008] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 22:03:27] "[36mGET /openapi/swagger/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:03:27,036] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 22:03:27] "[36mGET /openapi/swagger/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:03:27,038] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 22:03:27] "[36mGET /openapi/swagger/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:03:27,215] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 22:03:27] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:03:27,224] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 22:03:27] "[36mGET /openapi/swagger/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:03:51,053] ERROR log_exception() L875  Exception on /generate-rescue-point [POST] - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\flask\app.py L875 
Traceback (most recent call last):
  File "d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\flask_openapi3\scaffold.py", line 117, in view_func
    response = func(**func_kwargs)
  File "D:\Desenvolvimento\PUC\rescue-api\app.py", line 153, in generate_random_rescue_points
    rescue_points = generate_rescue_points(int(form.number))
  File "D:\Desenvolvimento\PUC\rescue-api\model\rescue_point.py", line 55, in generate_rescue_points
    latitude= rescue_point[0],
              ~~~~~~~~~~~~^^^
TypeError: 'int' object is not subscriptable
[2025-03-22 22:03:51,382] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 22:03:51] "[35m[1mPOST /generate-rescue-point HTTP/1.1[0m" 500 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:08:08,309] INFO _log() L97    * Detected change in 'D:\\Desenvolvimento\\PUC\\rescue-api\\model\\rescue_point.py', reloading - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:08:08,494] INFO _log() L97    * Restarting with stat - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:08:11,633] INFO _log() L97    * Detected change in 'D:\\Desenvolvimento\\PUC\\rescue-api\\model\\rescue_point.py', reloading - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:08:11,790] INFO _log() L97    * Restarting with stat - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:08:16,423] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 22:08:16] "[35m[1mPOST /generate-rescue-point HTTP/1.1[0m" 500 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 22:09:19,931] INFO _log() L97    * Restarting with stat - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
